src/components/About.js

```diff
@@ -10,8 +10,8 @@ function About() {
             <div className="row">
                 <div className="col-1-of-2 u-center-content u-increase-height">
                     <span className="headline section-subtitle">I'm a <span className="highlighted">full stack engineer</span> based in Phoenix, Arizona.</span>
-                    <p className="about-text">I have a passion for creating beautiful and accessible web and mobile applications. I believe the internet is for everyone, and accessibility-minded design makes it that way.</p>
-                    <p className="about-text">I'm proficient with the MERN stack, but learning new libraries and languages are a welcome challenge. As a developer my journey to learn is never over, I'm currently tackling NextJs and LottieFiles. </p>
+                    <p className="small-text">I have a passion for creating beautiful and accessible web and mobile applications. I believe the internet is for everyone, and accessibility-minded design makes it that way.</p>
```

Comment:
This typeface styling was something I wanted to be more 'global' - something that really does belong in Typography. It will be used for the <p> elements above the form, and other small text through out.